raiden/tests/integration/network/transport/rif_comms/test_client.py

```diff
@@ -23,92 +24,173 @@ def comms_nodes(amount_of_nodes) -> {int, CommsNode}:
 
 
 @pytest.mark.parametrize("amount_of_nodes", [1])
-def test_locate_own_peer_id(comms_nodes):
-    comms_node = comms_nodes[1]
+def test_subscribe_to_invalid(comms_nodes):
+    client = comms_nodes[1].client
+    # subscribe to unregistered address
+    topic_id, _ = client.subscribe_to(generate_address())
 
-    # attempt to locate self
-    assert comms_node.client._get_peer_id(comms_node.address) is not ""
+    # FIXME: this should probably throw an exception rather than an empty result
+    assert topic_id is ""
 
 
-# FIXME: causes ERR_NO_PEERS_IN_ROUTING_TABLE in comms node although it passes
 @pytest.mark.parametrize("amount_of_nodes", [1])
-def test_locate_unregistered_peer_id(comms_nodes):
-    comms_node = comms_nodes[1]
+def test_subscribe_to_self(comms_nodes):
+    client, address = comms_nodes[1].client, comms_nodes[1].address
+
+    topic_id, _ = client.subscribe_to(address)
+
+    assert topic_id
+
+
+@pytest.mark.parametrize("amount_of_nodes", [2])
+def test_subscribe_to_peers(comms_nodes):
+    comms_node_1, comms_node_2 = comms_nodes[1], comms_nodes[2]
+    client_1, address_1 = comms_node_1.client, comms_node_1.address
+    client_2, address_2 = comms_node_2.client, comms_node_2.address
+
+    topic_id_1, _ = client_1.subscribe_to(address_2)
```

Comment:
Same ignored value for the rest of the ocurrences

Reply:
> Same ignored value for the rest of the ocurrences

see [previous reply](https://github.com/rsksmart/lumino/pull/182#discussion_r546843587)